src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/ManageBookmarks/ManageBookmarks.xaml

```diff
@@ -0,0 +1,41 @@
+﻿<UserControl x:Class="ArcGISRuntime.Desktop.Samples.ManageBookmarks.ManageBookmarks"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013"
+             mc:Ignorable="d">
+    <Grid>
+
+        <esri:MapView x:Name="MyMapView" />
+        
+        <Border
```

Comment:
UI Suggestion: put all controls on a single line and use DisplayMemberPath to show the Name of the bookmark in the combo box (instead of adding just the bookmark names, e.g.). See comments for the codebehind files for more info.
````
<Border 
        Background="White" BorderBrush="Black" BorderThickness="1"
        HorizontalAlignment="Right" VerticalAlignment="Top"
        Margin="30" Padding="20" Width="330">
    <StackPanel Orientation="Horizontal">
        <TextBlock Text="Bookmarks:"
                        VerticalAlignment="Center"
                        Margin="5,0"/>
        <ComboBox x:Name="bookmarkChooser" 
                        DisplayMemberPath="Name" 
                        Width="160"
                        SelectionChanged="OnBookmarkChooserSelectionChanged" />
        <Button x:Name="ButtonAddBookmark"  
                    Margin="5,0" Padding="5"
                    Content="Add ..." 
                    Click="ButtonAddBookmark_Click"/>
    </StackPanel>
</Border>
````